tgstation.dme

```diff
@@ -738,7 +738,7 @@
 #include "code\game\world.dm"
 #include "code\game\area\ai_monitored.dm"
 #include "code\game\area\areas.dm"
-#include "code\game\area\Space_Station_13_areas.dm"
+#include "code\game\area\space_station_13_areas.dm"
```

Comment:
You forgot to rename the file 😛

Reply:
Man I totally did, vsc was freaking out over this all afternoon, but manually changing the name felt like a good way to break something 👀